lms/templates/components/card/card.underscore

```diff
@@ -2,40 +2,40 @@
 <div class="wrapper-card-core">
     <div class="card-core">
         <% if (pennant) { %>
-            <small class="card-type"><%- pennant %></small>
+            <small class="card-type"><%= HtmlUtils.escape(pennant) %></small>
         <% } %>
         <h3 class="card-title"
             <% if (!_.isUndefined(srInfo)) { %>
                 aria-describedby="<%= srInfo.id %>"
             <% } %>
-            ><%- title %>
+            ><%= HtmlUtils.escape(title) %>
```

Comment:
@andy-armstrong I spoke with Ned, and we are all in agreement, so we are going to go with my initial proposal.

That means that all of these changes should be using `<%-` unless you know it is a problem, in which case you will use the `escape()` method and possibly refactor names to `titleHtml` where appropriate.

Please confirm you are good with this.  Thanks!